oracle/oracle-how-to/use-bastion-to-access-VM.rst

```diff
@@ -1,25 +1,34 @@
 Use a bastion to access your VM
-===============================
+=============================== 
 
 To increase security and avoid Internet based attacks, it might make sense to run your VMs without a public IP address and just use a private IP instead. The only way to access such a VM will be through your private network in the Oracle Cloud VNC (Virtual Cloud Network).
 
-`Oracle's bastion feature`_ provides a time-limited access to VMs without a public endpoint. Bastions control authorised users and allow them to SSH into a VM from specific IP addresses. It also negates the need to maintain an instance with a public and private address to act as a bastion (which in turn runs the risk of Internet exposure).
+`Oracle's bastion feature`_ provides a time-limited access to VMs that do not have a public endpoint. Bastions control authorised users and allow them to SSH into a VM from specific IP addresses. It also negates the need to maintain an instance with a public and private address to act as a bastion (which in turn runs the risk of Internet exposure).
```

Comment:
"It" here should be either "They" or something more descriptive, such as "Use of a bastion"

Reply:
Updated it.